gossip/src/node.rs

```diff
@@ -122,80 +156,105 @@ impl Node {
                 &bind_ip_addrs,
                 tvu_port,
                 num_tvu_receive_sockets.get(),
-                socket_config,
+                socket_configs.primarily_read_udp,
             )
             .expect("Secondary bind TVU"),
         );
         let tvu_addresses = Self::get_socket_addrs(&tvu_sockets);
 
         let (tvu_quic_port, tvu_quic) =
-            bind_in_range_with_config(bind_ip_addr, port_range, socket_config)
+            bind_in_range_with_config(bind_ip_addr, port_range, socket_configs.primarily_read_quic)
```

Comment:
Technically same socket is used to send, but giving more buffers here is silly as it will just increase shred forwarding latency in a way we can not observe as stalls inside agave.